src/idpconfgen/cli_build.py

```diff
@@ -1258,6 +1266,26 @@ def conformer_generator(
                 # add the carboxyls
                 template_coords[TEMPLATE_MASKS.cterm] = \
                     MAKE_COORD_Q_COO_LOCAL(bb[-2, :], bb[-1, :])
+                    
+            except ValueError:
+                # Int2Cart requires length of seq to match length of torsions
+                # if there is a mismatch match is found, start from scratch
+
+                # Mismatches occur when the first residue of a sequence throws
+                # two sets of torsion angles, where Int2Cart requires only 1 set
+                # to correct on. For example:
+                '''Following example can be seen for Sic1
```

Comment:
Please change thins string in `'''` to a comment `#` because this code is executed millions of times and creating a string here may add non-negligible time delays to the protocol. After that you can merge. :+1:

Reply:
Oh I didn't know that! Good to know in the future. Thanks